tasks/src/visual-question-answering/about.md

```diff
@@ -38,3 +38,6 @@ vqa_pipeline(image, question, top_k=1)
 
 - [An introduction to Visual Question Answering - AllenAI](https://blog.allenai.org/vanilla-vqa-adcaaaa94336)
 - [Multi Modal Framework (MMF) - Meta Research](https://mmf.sh/docs/getting_started/video_overview/)
+
+The contents of this page are contributed by [
+Bharat Raghunathan](https://huggingface.co/bharat-raghunathan).
```

Comment:
Appreciate this! However, I'm not the only contributor to this page and @jlondonob and @mishig25 also deserve credit for this, please mention their hf usernames too if available